### PR TITLE
Make The Profiles Alphabetic Order

### LINKED ISSUE #661: Make The Profiles Alphabetic Order
the repo will look better if the profiles are in Alphabetic Order instead  of making it random

### PR DESCRIPTION
**Pull request by qwe123coder**
Resolve #661

### CODE REVIEW

**Comment by CodingSpecies** [comment]
@qwe123coder , looks good, did you use any method or anything to alphabetically change the whole file?

**Comment by qwe123coder** [comment]
yep a tool

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
> yep a tool

What tool did you used and is it possible to run a github action that automatically sorts the data.json file?

**Comment by qwe123coder** [comment]
i used this https://novicelab.org/jsonabc/

**Comment by qwe123coder** [comment]
but 1 problem it even takes numbers first and then only alphabets

**Comment by qwe123coder** [comment]
i could try to make the same thing in github action

File: data.json [hunk 1]
```diff
@@ -1,1286 +1,1287 @@
...
+        "name": "Mohamed Yaseen Sheriff S"
+    },
+    {
+        "githubUsername": "YashIndane",
+        "imageUrl": "https://user-images.githubusercontent.com/53041219/99989860-22353e80-2dd9-11eb-95c5-07dd04eecd7d.png",
+        "issueNumber": 180,
+        "name": "Yash Indane"
+    },
+    {
+        "githubUsername": "aakarshteja",
```
Comment (Panquesito7) [inline data.json:526]: Why are these not in the first lines? They start with `a` (same in other parts of the file).

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data.json [hunk 2]
```diff
@@ -1,1286 +1,1287 @@
...
-  },
-  {
-    "name": "Anik Chakraborty",
-    "githubUsername": "Anikcb",
-    "imageUrl": "https://user-images.githubusercontent.com/47493387/130086990-1072a097-f8b7-423b-8eeb-b161870b11ea.png",
-    "issueNumber": 657
-  }
+
+    {
+        "githubUsername": "ATCtech",
```
Comment (KukretiShubham) [inline data.json:4]: I guess, there used to be 'name' first.
Save goes for everyone. 
won't there be an issue with that?
```suggestion
        "name": "Ankush Singh",
```

File: data.json [hunk 3]
```diff
@@ -1,1286 +1,1287 @@
...
+        "imageUrl": "https://user-images.githubusercontent.com/50258860/94898939-046dec00-04b0-11eb-8a6a-10a0c59da3f0.JPG",
+        "issueNumber": 115,
+        "name": "Ankush Singh"
```
Comment (KukretiShubham) [inline data.json:7]:
```suggestion
        "githubUsername": "ATCtech"
```

**Review by KukretiShubham — COMMENTED** [review]
I guess, there used to be 'name' first.
Save goes for everyone.
won't there be an issue with that?

File: data.json (hunk 1 above)
Comment (qwe123coder) [reply data.json:526]: oh that it is done by names and not by git-hub username

**Comment by KukretiShubham** [comment]
@qwe123coder My suggestion was for the entire file. Whole file has same issue. May be this is the reason, checks are failing.

**Comment by qwe123coder** [comment]
i do not think it is because of that 
image

**Comment by KukretiShubham** [comment]
oh, but then you will have to change github actions accordingly as order is now being changed. NVM I will also look for alt. 🚀

**Comment by Panquesito7** [comment]
> i do not think it is because of that
> image

This doesn't work because you deleted this branch.

**Pull request closed without merging**